src/mpid/ch4/shm/posix/lmt/rndv/rndv_control.c

```diff
@@ -0,0 +1,201 @@
+/*
+ *  (C) 2019 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ *  Portions of this code were written by Intel Corporation.
+ *  Copyright (C) 2011-2019 Intel Corporation.  Intel provides this material
+ *  to Argonne National Laboratory subject to Software Grant and Corporate
+ *  Contributor License Agreement dated February 8, 2012.
+ */
+
+#include "rndv.h"
+
+/* Handle incoming ready to send (RTS) message by setting up a shared memory segment and enqueuing
+ * the message into a list to progressed.
+ *
+ * ctrl_hdr - The struct containing the control message
+ */
+int MPIDI_POSIX_lmt_ctrl_send_lmt_msg_cb(MPIDI_SHM_ctrl_hdr_t * ctrl_hdr)
+{
```

Comment:
Can we use `rts|cts` instead of `msg|ack`? The XPMEM code also uses `rts|cts`.

Reply:
Done.